llvm/runtimes/CMakeLists.txt

```diff
@@ -269,11 +270,12 @@ endfunction()
 # runtime_register_target(name)
 #   Utility function to register external runtime target.
 function(runtime_register_target name)
-  cmake_parse_arguments(ARG "" "BASE_NAME" "DEPENDS;CMAKE_ARGS;EXTRA_ARGS" ${ARGN})
+  cmake_parse_arguments(ARG "" "BASE_NAME" "DEPENDS;CMAKE_ARGS;EXTRA_ARGS;RUNTIMES" ${ARGN})
   include(${LLVM_BINARY_DIR}/runtimes/${name}/Components.cmake OPTIONAL)
   set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${LLVM_BINARY_DIR}/runtimes/${name}/Components.cmake)
 
   set(runtime_names ${RUNTIME_NAMES})
+  set(LLVM_ENABLE_RUNTIMES ${ARG_RUNTIMES})
```

Comment:
It's confusing to set this variable here since it may imply that it is somehow related to the loop below while it isn't, this should be moved to line 341.

Reply:
Good point, done.